src/generators/clientFileGenerator.ts

```diff
@@ -54,16 +59,18 @@ export function generateClient(clientDetails: ClientDetails, project: Project) {
       moduleSpecifier: "@azure/core-http"
     });
 
+  if (hasImportedOperations) {
```

Comment:
Included this fix as I found an error where the SDK had both Client level operations and operation group classes, we were not importing the Operations